### PR TITLE
Render link to profiling report in console

### PR DESCRIPTION
**Pull request by bmuschko**
PR for https://github.com/gradle/gradle/issues/1354

### CODE REVIEW

**Comment by eriwen** [comment]
Should we add this to the release notes?

File: subprojects/core/src/main/java/org/gradle/profile/ReportGeneratingProfileListener.java [hunk 1]
```diff
@@ -35,6 +44,14 @@ public void buildFinished(BuildProfile buildProfile) {
         ProfileReportRenderer renderer = new ProfileReportRenderer();
         File file = new File(buildDir, "reports/profile/profile-" + FILE_DATE_FORMAT.format(new Date(buildProfile.getBuildStarted())) + ".html");
         renderer.writeTo(buildProfile, file);
+        renderReportUrl(file);
+    }
+
+    private void renderReportUrl(File reportFile) {
+        StyledTextOutput textOutput = textOutputFactory.create(ReportGeneratingProfileListener.class, LogLevel.LIFECYCLE);
+        textOutput.println();
```
Comment (eriwen) [inline subprojects/core/src/main/java/org/gradle/profile/ReportGeneratingProfileListener.java]: Based on the output I see, we may not need this extra println.

screen shot 2017-02-17 at 9 21 02 pm

**Comment by eriwen** [comment]
Just a couple trivial nitpicks. Otherwise LGTM.

File: subprojects/core/src/main/java/org/gradle/profile/ReportGeneratingProfileListener.java (hunk 1 above)
Comment (bmuschko) [inline subprojects/core/src/main/java/org/gradle/profile/ReportGeneratingProfileListener.java]: I think the offender is actually the Build Scan plugin that seems to add a new line at the end. If you do not apply the Build Scan plugin then the output looks proper.

**Comment by bmuschko** [comment]
To be honest I don't think it's worth it to add something for this in the release notes. No sure what I would write: The profile report now renders a clickable link to the report on the console? IMHO that's covered by the actual issue that will be listed in the release notes anyway.

**Pull request merged**